### PR TITLE
Add page disclaimers

### PR DESCRIPTION
**Pull request by nerik**
Adds the possibility to open a disclaimer modal upon entering a given section of the dashboard; currently Analysis and  Dataset Explore. Uses a localStorage item to prevent the disclaimers to show up again if the user don't want to (which is the default choice)

### CODE REVIEW

File: overrides/components/page-footer/component.tsx [hunk 1]
```diff
@@ -92,102 +95,227 @@ const TintBox = styled.div`
...
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row nowrap;
+  margin-top: ${glsp(2)};
+`;
+
+const MODALS_DISMISSED_KEY = "modalsDismissedKey";
+
+const MODALS_CONTENT = {
+  explore: {
```
Comment (hanbyul-here) [inline overrides/components/page-footer/component.tsx:116]: We can use the routes defined in https://github.com/NASA-IMPACT/veda-ui/blob/main/app/scripts/utils/routes.ts?

File: overrides/components/page-footer/component.tsx [hunk 2]
```diff
@@ -92,102 +95,227 @@ const TintBox = styled.div`
...
+`;
+
+const DisclaimerModalFooter = styled(ModalFooter)`
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row nowrap;
+  margin-top: ${glsp(2)};
+`;
+
+const MODALS_DISMISSED_KEY = "modalsDismissedKey";
```
Comment (hanbyul-here) [inline overrides/components/page-footer/component.tsx:113]: prefix the key name since it is pretty generic?

**Review by hanbyul-here — COMMENTED** [review]
This seems a better approach than touching the ui code. I left two comments

**Review by j08lue — CHANGES_REQUESTED** [review]
It seems like the modal on Analysis won't show if I clicked the one on Exploration away, and vice versa.

If that is the case, we need to change that, please - the modals are different on each tool and the user needs to acknowledge them all at least once, please.

**Comment by nerik** [comment]
@j08lue Good catch. There was an issue when switching from one page to another. Should be fixed now.

**Review by hanbyul-here — APPROVED** [review]
Ah I didn't notice that we don't have explore path exported from ui module. it looks good to me.

**Pull request merged**